src/AppInstallerSharedLib/AppInstallerLogging.cpp

```diff
@@ -160,24 +160,27 @@ namespace AppInstaller::Logging
     }
 }
 
-std::ostream& operator<<(std::ostream& out, const std::chrono::system_clock::time_point& time)
+namespace std
```

Comment:
Adding to the `std` namespace seems weird

Reply:
I don't know what properties the latest C++/WinRT is forcing on us, but they ended up breaking most of these `operator<<` overloads that were in the global namespace.  Leveraging [ADL](https://en.cppreference.com/w/cpp/language/adl) to overcome this requires putting these into `std` (or `std::chrono` here).